src/main/index.js

```diff
@@ -855,8 +861,36 @@ function runApp() {
     session.defaultSession.closeAllConnections()
   })
 
-  ipcMain.on(IpcChannels.OPEN_EXTERNAL_LINK, (_, url) => {
-    if (typeof url === 'string') shell.openExternal(url)
+  ipcMain.handle(IpcChannels.OPEN_EXTERNAL_LINK, (_, url) => {
+    if (typeof url === 'string') {
+      let parsedURL
+
+      try {
+        parsedURL = new URL(url)
+      } catch {
+        // If it's not a valid URL don't open it
+        return false
+      }
+
+      if (
+        parsedURL.protocol === 'http:' || parsedURL.protocol === 'https:' ||
+
+        // Email address on the about page and Autolinker detects and links email addresses
+        parsedURL.protocol === 'mailto:' ||
+
+        // Autolinker detects and links phone numbers
+        parsedURL.protocol === 'tel:' ||
+
+        // Donation links on the about page
+        (parsedURL.protocol === 'bitcoin:' && parsedURL.pathname === ABOUT_BITCOIN_ADDRESS) ||
+        (parsedURL.protocol === 'monero:' && parsedURL.pathname === ABOUT_MONERO_ADDRESS)
+      ) {
+        shell.openExternal(url)
+        return true
+      }
+    }
+
+    return false
```

Comment:
Nitpick on code style (intent)
The smaller code block should be nested instead of the bigger one
Change it this PR if you want but I notice this in other places too so it's more like a reminder/rant
```js
if (typeof url !== 'string') { return false }

// Rest of bigger code block
```